### ISSUE TITLE
could example how to manage app structure for multi module (multi app feature)?

### ISSUE THREAD

**Issue by thearabbit** [chris-visser/meteor-vue-admin#2]
Example I wold like to manage multi app feature on the same source
- Accounting module
- Loan module
- Saving Module

Could example app structure?

**Comment by thearabbit** [comment]
I mean that I have `core app` and have sub systems in `core` like this
```js
/app
   import/
       //--- core
       api/
       startup/
       client/
       //--- modules
       modules/
             accounting/
                     api/
                     startup/
                     client/
              pos/
                     api/
                     startup/
                     client/
```

**Comment by chris-visser** [comment]
Yes this should be on the list. I will update the TODO's because modules will be unavoidable. Good one 🙂

**Comment by thearabbit** [comment]
thanks for your reply, 💯

**Comment by chris-visser** [comment]
@thearabbit how would you see this multi site thing? As in: Each module represents a certain frontend? Or are you referring to a feature based system where for example "auth" and "articles" would be a feature? Am a bit confused right now

**Comment by thearabbit** [comment]
I would like to manage multi module system like `Odoo System`:
(Each module represents a main layout)
- Welcome page
![image](https://user-images.githubusercontent.com/4495160/50407724-a6f33880-080f-11e9-8fee-c4d1b0518af3.png)
- I try to access `Point of Sale`
![image](https://user-images.githubusercontent.com/4495160/50407734-c1c5ad00-080f-11e9-9b8b-6842efd7f983.png)

(Sorry if I am confused your question)

**Comment by chris-visser** [comment]
No worries. We'll figure it out. I'm refurbishing a bit to accommodate a feature based structure. My key time consumer here is to keep it clear what I did and how things are connected

**Comment by chris-visser** [comment]
Done :+1:

**Comment by thearabbit** [comment]
very thanks 💯
